grpc/creds/creds_test.go

```diff
@@ -82,11 +84,16 @@ func TestTransportCredentials(t *testing.T) {
 		_ = ln.Close()
 	}()
 	addrC := ln.Addr().String()
+	stop := make(chan struct{}, 1)
 	go func() {
 		for {
-			_, err := ln.Accept()
-			test.AssertNotError(t, err, "ln.Accept failed")
-			time.Sleep(time.Second)
+			select {
+			case <-stop:
+				return
+			default:
+				_, _ = ln.Accept()
+				time.Sleep(2 * time.Millisecond)
```

Comment:
The  `time.Sleep` being removed slept for 1s after the `ln.Accept()`. Is `2 * time.Millisecond` an OK replacement?